lib/tessel/wifi.js

```diff
@@ -60,9 +64,10 @@ Tessel.prototype.getWifiInfo = function() {
 };
 
 function compareBySignal(a, b) {
-  if ((a.quality / a.quality_max) > (b.quality / b.quality_max)) {
+  // eval(quality) turns a string fraction like 21/70 into a decimal
+  if (eval(a.quality) > eval(b.quality)) {
     return -1;
-  } else if ((a.quality / a.quality_max) < (b.quality / b.quality_max)) {
+  } else if (eval(a.quality) < eval(b.quality)) {
```

Comment:
```javascript
function parseFraction (src) {
  var m = /([0-9]+)\/([0-9]+)/.exec(src)
  if (m === null) return NaN

  return Number(m[1]) / Number(m[2])
}
```

Maybe it's not that big of a security hazard in this particular instance, but I would consider it a code smell. It's easy to parse without `eval`, and it makes the intent clearer.